File: src/test/java/br/com/model/entities/classes/NovoClienteTest.java
package br.com.model.entities.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.entities.classes.usuario.NovoCliente;

public class NovoClienteTest {
    @Test
    public void novoClienteTeste() {
        var cliente = NovoCliente.novoCliente();
        System.out.println(cliente.toString());
    }
}

File: src/test/java/br/com/model/entities/classes/NovoClienteTest.kt
package br.com.model.entities.classes

import br.com.newObjects.model.entities.classes.usuario.NovoCliente
import org.junit.jupiter.api.Test

class NovoClienteTest {
    @Test
    fun novoClienteTeste() {
        val cliente = NovoCliente.novoCliente()
        println(cliente.toString())
    }
}
